sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/FeedResponseDiagnostics.java

```diff
@@ -4,26 +4,30 @@
 package com.azure.cosmos.implementation;
 
 import com.azure.cosmos.implementation.query.QueryInfo;
-import com.azure.cosmos.implementation.query.metrics.QueryMetricsTextWriter;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
+import com.fasterxml.jackson.datatype.jsr310.deser.DurationDeserializer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.time.Duration;
+import java.time.Instant;
 import java.util.Collection;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
+import static com.azure.cosmos.implementation.guava27.Strings.lenientFormat;
+
 /**
  * The type Feed response diagnostics.
  */
 public class FeedResponseDiagnostics {
 
-    private final static String EQUALS = "=";
-    private final static String QUERY_PLAN = "QueryPlan";
-    private final static String SPACE = " ";
     private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
On second thought, I am curious if we can re-use the `Utils.getSimpleObjectMapper()`?

Reply:
Utils.getSimpleObjectMapper might be used in multiple places. And we have added additional serialization and deserialization config. Hence preferred to keep it custom within this class.

Reply:
I've added another helper method in the Utils to create an object mapper with duration configs enabled and using that now